src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -272,4 +299,8 @@ public VirtualHost build() {
     public String toString() {
         return VirtualHost.toString(getClass(), defaultHostname, hostnamePattern, sslContext, services);
     }
+
+    public ServerBuilder and() {
+        return serverBuilder;
```

Comment:
What would this method be supposed to do when `serverBuilder` is `null`? It seems to me that this method cannot do its job when `VirtualHostBuilder` was instantiated via the constructors that do not require `ServerBuilder`.